src/platform/loader-base.ts

```diff
@@ -24,6 +24,7 @@ import {logsFactory} from '../platform/logs-factory.js';
 import {Dictionary} from '../runtime/hot.js';
 import '../runtime/schema-from-literal.js';
 import '../runtime/type-from-literal.js';
+import '../runtime/handle-constructors.js';
```

Comment:
Could we list the imports? Otherwise we don't know what we're importing (the others have this problem, but ideally we wouldn't make it worse)

Reply:
There are no imports. The files are being loaded solely to run the code on loading. I'll add a comment.